.github/workflows/clean-pr.yaml

```diff
@@ -12,8 +12,8 @@ on:
         description: 'Pull request number'
 
 jobs:
-  cleanup:
-    name: Clean up cache
+  cleanup-pr:
+    name: Clean up PR cache
```

Comment:
Just from the visual perspective, this is what will the job look like on the Github tab:
`Cleanup PR | Clean up PR cache`

I think the job name could be:

```suggestion
    name: Remove cache
```

To yield:
`Cleanup PR | Remove cache`